tools/specs/Microsoft.Data.SqlClient.nuspec

```diff
@@ -72,6 +73,7 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
         <dependency id="Microsoft.Identity.Client" version="4.22.0" exclude="Compile"/>
         <dependency id="Microsoft.IdentityModel.Protocols.OpenIdConnect" version="6.8.0" />
         <dependency id="Microsoft.IdentityModel.JsonWebTokens" version="6.8.0" />
+        <dependency id="System.Configuration.ConfigurationManager" version="4.7.0" exclude="Compile" />
```

Comment:
[ConfigurationManager](https://docs.microsoft.com/en-us/dotnet/api/system.configuration.configurationmanager?view=net-5.0) doesn't support net standard.
```suggestion
```

Reply:
That might be doc issue.

Here it shows it does: https://www.nuget.org/packages/System.Configuration.ConfigurationManager/4.7.0